cypress/e2e/smoke/address_book.cy.js

```diff
@@ -69,22 +69,17 @@ describe('Address book', () => {
       cy.contains('button.MuiButton-sizeMedium', 'Import').click()
       cy.findByText(RINKEBY_CSV_ENTRY.name).should('exist')
       cy.findByText(RINKEBY_CSV_ENTRY.address).should('exist')
+      cy.wait(1000)
     })
 
-    it.skip('should find Gnosis Chain imported address', () => {
-      //Test skipped because it causes an error in the simulation
-      //"Unhandled Runtime Error
-      //Error: Hydration failed because the initial UI does not match what was rendered on the server."
-
+    it('should find Gnosis Chain imported address', () => {
```

Comment:
This test is failing for me (in Chrome):

![image](https://user-images.githubusercontent.com/20442784/187836541-6f906914-bb0c-4c1e-806d-70ed014caa91.png)

Reply:
This test is supposed to check that the .csv (that has 2 addresses for rinkeby and 2 for gnosis) was properly imported. The test switches to the GNO chain and checks the address book there for the addresses. For some reason sometimes the import works fine and the addresses are there and other times those aren't there.

Reply:
This part of the test was skipped before. I can set it to be skipped again if it fails randomly.